tests/fixtures_new.py

```diff
@@ -0,0 +1,240 @@
+import pytest
+from tests import helpers
+from ocs_ci.ocs import constants, ocp
+
+
+@pytest.fixture()
+def secret(request, interface_type):
```

Comment:
is interface_type other fixture? Or it's parameter here? If it's parameter I don't think you can pass parameter to fixture like this. You will have to create fixture factory to have this capability

Reply:
Yeah, the only way I know of to allow fixtures to use parameters is to use the factory approach.